xarm_moveit_config/launch/_robot_moveit_common.launch.py

```diff
@@ -167,18 +159,19 @@ def launch_setup(context, *args, **kwargs):
         # },
     }
 
-    # sensor_manager_parameters = {
-    #     'sensors': ['ros'],
-    #     'octomap_resolution': 0.02,
-    #     'ros.sensor_plugin': 'occupancy_map_monitor/PointCloudOctomapUpdater',
-    #     'ros.point_cloud_topic': '/camera/depth/color/points',
-    #     'ros.max_range': 2.0,
-    #     'ros.point_subsample': 1,
-    #     'ros.padding_offset': 0.1,
-    #     'ros.padding_scale': 1.0,
-    #     'ros.max_update_rate': 1.0,
-    #     'ros.filtered_cloud_topic': 'filtered_cloud',
-    # }
+    sensor_manager_parameters = {
```

Comment:
Can we read this from the yaml file instead?

Reply:
Yes, we should put it in yaml that is usually the convention